tests/components/remote/test_init.py

```diff
@@ -13,109 +20,128 @@
     STATE_ON,
 )
 
-from tests.common import get_test_home_assistant, mock_service
-from tests.components.remote import common
+from tests.common import async_mock_service
 
-TEST_PLATFORM = {remote.DOMAIN: {CONF_PLATFORM: "test"}}
+TEST_PLATFORM = {DOMAIN: {CONF_PLATFORM: "test"}}
 SERVICE_SEND_COMMAND = "send_command"
 SERVICE_LEARN_COMMAND = "learn_command"
+SERVICE_DELETE_COMMAND = "delete_command"
+ENTITY_ID = "entity_id_val"
+
+
+async def test_is_on(hass):
+    """Test is_on."""
+    hass.states.async_set("remote.test", STATE_ON)
+    assert remote.is_on(hass, "remote.test")
+
+    hass.states.async_set("remote.test", STATE_OFF)
+    assert not remote.is_on(hass, "remote.test")
+
+
+async def test_turn_on(hass):
+    """Test turn_on."""
+    turn_on_calls = async_mock_service(hass, DOMAIN, SERVICE_TURN_ON)
+    await hass.services.async_call(DOMAIN, SERVICE_TURN_ON, {ATTR_ENTITY_ID: ENTITY_ID})
+
+    await hass.async_block_till_done()
+
+    assert len(turn_on_calls) == 1
+    call = turn_on_calls[-1]
+
+    assert DOMAIN == call.domain
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6356951)